docs/content/en/docs/core-components/api-manual.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Manual API Description
+description: >
+    Descriptions for the SOARCA manual interaction REST API endpoints 
+categories: [API]
+tags: [protocol, http, rest, api]
+weight: 3
+date: 2024-05-21
+---
+
+## Endpoint descriptions
+
+We will use HTTP status codes https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
+
+
+```plantuml
+@startuml
+protocol Reporter {
+    GET     /manual
+    POST    /manual/continue
+}
+@enduml
+```
+
+
+### /manual
+The manual interaction endpoint for SOARCA
+
+#### GET `/manual`
+Get all pending manual actions objects that are currently waiting in SOARCA.
+
+##### Call payload
+None
+
+##### Response
+200/OK with payload:
+
+
+
+|field              |content                |type               | description |
+| ----------------- | --------------------- | ----------------- | ----------- |
+|type               |execution-status       |string             |The type of this content
+|execution_id       |UUID                   |string             |The id of the execution
+|playbook_id        |UUID                   |string             |The id of the CACAO playbook executed by the execution
+|step_id            |UUID                   |string             |The id of the step executed by the execution
+|description        |description of the step|string             |The description from the workflow step
+|command            |command                |string             |The command for the agent either command 
+|command_is_base64  |true \| false          |bool               |Indicate the command is in base 64
+|targets            |cacao agent-target     |dictionary         |Map of [cacao agent-target](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256509) with the target(s) of this command
+|in_args            |cacao variables        |dictionary         |Map of [cacao variables](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256555) handled in the step in args with current values and definitions
```

Comment:
I would discurage in any case the use and reporting of in_args, as I don't think there's really any way to use them. They should be already resolved in the `command` if any